menyer/ラーメン/index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;1,300&display=swap">
+    <link rel="stylesheet" href="style.css">
+    <title>ラーメン</title>
+</head>
+
+<body>
+    <header>
+        <article>
+            <img src="img/img_header_1.png" alt="bowle of ramen" width="400px">
+            <span>
+                <h1>ラーメン</h1>
+            </span>
+        </article>
+    </header>
+    <main>
+        <article>
+            <div class="distance">
```

Comment:
Class names should convey meaning. I am not sure what `distance` means.

Reply:
If you want to have distance between articles I suggest that you add a margin to your article.